sinks/redis/producer/events.go

```diff
@@ -49,12 +49,14 @@ func (cce createSinkEvent) Encode() map[string]interface{} {
 }
 
 type deleteSinkEvent struct {
-	id string
+	sinkID    string
+	token string
```

Comment:
we don't want to send tokens through redis events ... we don't need to because the internal events are considered a secure channel. instead we should send whatever information is needed on the other side (ownerID, etc) and call an "Internal" service function which don't take a token (see the existing examples). so we probably want `InactivateDatasetByIDInternal` for example, which will never be called by e.g. the REST API. let me if we need to discuss this to make it clearer.

Reply:
Make sense, got it!